components/tree/tree-drop-indicator.component.ts

```diff
@@ -20,8 +29,8 @@ import { NgStyleInterface } from 'ng-zorro-antd/core/types';
   standalone: true
 })
 export class NzTreeDropIndicatorComponent implements OnChanges {
-  @Input() dropPosition?: number;
-  @Input() level: number = 1;
+  @Input({ transform: numberAttributeWithZeroFallback }) dropPosition?: number;
```

Comment:
this fallback will lead to different behavior

<img width="676" alt="image" src="https://github.com/NG-ZORRO/ng-zorro-antd/assets/49607541/94cf27ff-aeff-4a50-a54d-cc9c848aab5f">